content/03.results.md

```diff
@@ -194,18 +198,9 @@ Additionally, we found a significant positive correlation between _TP53_ scores
 Tumors with the highest median _TP53_ scores were those known to harbor somatic _TP53_ alterations: choroid plexus tumors, embryonal tumors, HGATs, and pineal tumors (**Figure {@fig:Fig5}F**), while melanocytic tumors, meningiomas, and tumors of cranial and paraspinal nerves had the lowest scores.
 
 We next used gene expression data to predict telomerase activity using EXpression-based Telomerase ENzymatic activity Detection (EXTEND) [@doi:10.1038/s41467-020-20474-9] as a surrogate measure of malignant potential [@doi:10.1038/s41467-020-20474-9; 10.1093/carcin/bgp268].
-EXTEND scores significantly correlated with _TERT_ and _TERC_ expression (**Figure {@fig:FigS5}C-D**).  
+EXTEND scores significantly correlated with _TERT_ and _TERC_ expression (**Figure {@fig:FigS5}G-H**).  
```

Comment:
I'd report the correlations here. I suggest this because we really do have a meaningful effect size here, don't bury it! (Whenever I see a paper report significant without giving the effect size, I assume it's because the effect size is not meaningful and they are hiding it...).
```suggestion
EXTEND scores significantly correlated with _TERT_ (r=0.55) and _TERC_ (r=0.58) expression (**Figure {@fig:FigS5}G-H**).  
```